### PR TITLE
Update qhimm.xml

### PR DESCRIPTION
**Pull request by bradhowlett**
Here is my retry on doing this correct. The only thing I don't quite understand is the whole iros/GDrive linking methodology.

### CODE REVIEW

File: catalogs/qhimm.xml [hunk 1]
```diff
@@ -2222,4 +2222,37 @@ Everyone has a Curse Ring Equipped, the challenge is to win before Death Sentanc
     </Mod>
 
   </Mods>
+
+## WereXIII - by Wolfman
+    <Mod>
+      <ID>1iwfYEec-eu6K4hB6zD06dok1jH92lX3n</ID>
+      <Author>Tsunamods</Author>
+      <Link>https://drive.google.com/file/d/1iwfYEec-eu6K4hB6zD06dok1jH92lX3n/view?usp=drive_link</Link>
```
Comment (julianxhokaxhiu) [inline catalogs/qhimm.xml:2230]: The link on this tag should be like where you want your users need to go to know more about your mod, not the download link.

File: catalogs/qhimm.xml [hunk 2]
```diff
@@ -2222,4 +2222,37 @@ Everyone has a Curse Ring Equipped, the challenge is to win before Death Sentanc
...
+      <LatestVersion>
+        <Link></Link>
```
Comment (julianxhokaxhiu) [inline catalogs/qhimm.xml:2232]: Here instead the download link should be put, but instead with this syntax: `iros://GDrive/ID` where ID is the unique UUID drive gives you after `/d/`

File: catalogs/qhimm.xml [hunk 3]
```diff
@@ -2222,4 +2222,37 @@ Everyone has a Curse Ring Equipped, the challenge is to win before Death Sentanc
...
+        <Version>3.0</Version>
+        <ReleaseDate>2014-06-10</ReleaseDate>
+        <CompatibleGameVersions>All</CompatibleGameVersions>
+        <PreviewImage></PreviewImage>
+        <ReleaseNotes/>
+        <DownloadSize>419</DownloadSize>
```
Comment (julianxhokaxhiu) [inline catalogs/qhimm.xml:2238]: This should be in KB ( just open up an explorer window in Windows and put the same number it gives you in KB )

**Review by julianxhokaxhiu — CHANGES_REQUESTED** [review]
Minor changes required but you're on the right track. Please do no open another PR, just update this one instead. Thank you!

**Comment by bradhowlett** [comment]
> Minor changes required but you're on the right track. Please do no open another PR, just update this one instead. Thank you!

I cannot find where to go to edit my request?

**Comment by bradhowlett** [comment]
> This could be one way: https://docs.github.com/en/codespaces/the-githubdev-web-based-editor#opening-the-githubdev-editor

Ok. i believe I fixed it. I am not 100% positive my link to my file and my ID are correct but I am sure you will let me know

**Comment by bradhowlett** [comment]
Sorry I accidentally created another pull but I closed it

**Comment by julianxhokaxhiu** [comment]
Closing in favor of #207 - Your changes have been reflected there, not here. But please next time, find some time to learn a bit how to contribute properly :) Thank you

**Pull request closed without merging**